packages/manager/modules/vps/src/dashboard/vps-dashboard.controller.js

```diff
@@ -510,4 +510,11 @@ export default class {
   static getActionStateParamString(params) {
     return params ? `(${JSON.stringify(params)})` : '';
   }
+
+  canAdditionalDiskBeTerminated() {
```

Comment:
Looks like this logic is duplicated, maybe we could expose a utility `canOptionBeterminated` instead

Reply:
Done. https://github.com/ovh/manager/commit/62be96d5e74509e9194ccf9680e478b8a5a35249